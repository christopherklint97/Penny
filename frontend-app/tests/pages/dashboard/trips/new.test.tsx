import React from 'react'
import { render } from '../../../testUtils'
import NewTrip from '../../../../pages/dashboard/trips/new'

describe('NewTrip page', () => {
  it('renders without crashing', function () {
    render(<NewTrip />)
  })

  it('matches snapshot', () => {
    const { asFragment } = render(<NewTrip />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
