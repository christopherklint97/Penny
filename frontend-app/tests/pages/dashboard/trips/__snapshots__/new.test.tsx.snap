// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewTrip page matches snapshot 1`] = `<DocumentFragment />`;
